packages/jsapi-components/src/spectrum/Picker/Picker.tsx

```diff
@@ -6,13 +6,16 @@ import {
 import { useApi } from '@deephaven/jsapi-bootstrap';
 import { dh as DhType } from '@deephaven/jsapi-types';
 import { Formatter } from '@deephaven/jsapi-utils';
+import Log from '@deephaven/log';
 import { PICKER_ITEM_HEIGHT, PICKER_TOP_OFFSET } from '@deephaven/utils';
 import { useCallback, useEffect, useMemo } from 'react';
 import useGetItemIndexByValue from '../../useGetItemIndexByValue';
 import { useViewportData } from '../../useViewportData';
 import { getPickerKeyColumn } from './PickerUtils';
 import { usePickerItemRowDeserializer } from './usePickerItemRowDeserializer';
 
+const log = Log.module('Picker');
```

Comment:
One problem with log API is it won't disambiguate between any other `'Picker'` components. Could ensure it's "unique" by prefixing it with `jsapi-components`... doesn't matter much to me.